### PR TITLE
Remove the set_version and set_variant methods

### PR DESCRIPTION
**Pull request by kinggoesgaming**
**I'm submitting a(n)** removal

# Description
Removes `Uuid::set_variant` and `Uuid::set_version`

# Motivation
These methods were private and already `uuid::builder::Builder` was being recommended.

# Tests
Current tests should pass.

# Related Issue(s)
N/A

### CODE REVIEW

**Comment by kinggoesgaming** [comment]
depends on #399

**Comment by kinggoesgaming** [comment]
Ignore restyled for me

File: src/lib.rs [hunk 1]
```diff
@@ -956,6 +940,7 @@ mod tests {
             Uuid::new_v3(&Uuid::NAMESPACE_DNS, "rust-lang.org".as_bytes());
 
         assert_eq!(uuid.get_version().unwrap(), Version::Md5);
+        println!("pass1");
```
Comment (KodrAus) [inline src/lib.rs]: We should remove this before merging

Comment (kinggoesgaming) [reply src/lib.rs]: um... yea... I made a mistake during the refactor and couldn't figure out where was fail happening (cuz there are multiple asserts)

doing it rn

**Pull request merged**